src/chia_log/log_consumer.py

```diff
@@ -50,53 +54,69 @@ def _notify_subscribers(self, logs: str):
 
 
 class FileLogConsumer(LogConsumer):
-    """Specific implementation for a simple file consumer"""
-
     def __init__(self, log_path: Path):
-        logging.info("Enabled file log consumer.")
         super().__init__()
         self._log_path = log_path
+        self._expanded_log_path = ""
```

Comment:
Can we initialise this immediately?
```
str(self._log_path.expanduser())
```
It seems then we also don't need to keep `self._log_path` in the state.